pkg/dag/dag_uki_boot.go

```diff
@@ -60,8 +63,12 @@ func RegisterUKI(s *state.State, g *herd.Graph) error {
 	// Depends on mount binds as that usually mounts COS_PERSISTENT
 	s.LogIfError(s.MountCustomBindsDagStep(g, herd.WeakDeps), "custom binds mount")
 
+	// Copy any sysextensions found under cnst.SourceSysExtDir into cnst.DestSysExtDir so its loaded by systemd automatically on start
+	// always after cnst.OpMountBind stage so we have a persistent cnst.DestSysExtDir
+	s.LogIfError(s.CopySysExtensionsDagStep(g, herd.WithDeps(cnst.OpMountBind)), "copy sysextensions")
```

Comment:
We may not need this as long as we tell sysext that we are on initrd, it will go over the .extra dir itself